react/src/components/Boxes.module.css

```diff
@@ -1,10 +1,25 @@
+/* Universal Layout */
+:root {
+    /* Descriptive variables */
+    --color-blue: #428D9E;
+    --color-red: #c83434;
+    --color-yellow: #dfbb42;
+    --color-green: #508b61;
+
+    /* Functional variables */
+    --color-topLeftBox: var(--color-blue);
+    --color-topRightBox: var(--color-red);
+    --color-bottomLeftBox: var(--color-yellow);
+    --color-bottomRightBox: var(--color-green);
+    --color-centerBoxText: var(--color-darkGray);
+}
```

Comment:
https://www.joshwcomeau.com/css/css-variables-for-react-devs/

Do we have to declare `--color-*` variables in every file?

Reply:
Agh, for some reason I thought `:root` was equivalent to "this component only"...

Either way, the idea was to make the single-use variables local, but more general, instead of global, but more specific (App.css already has the intended global, multi-use variables).

For instance, both `Boxes` and `CarouselTopSection` (supposedly) have a local `--color-green` variable. The one in `Boxes` has the hex value of `#508b61`, while the one in `CarouselTopSection` has the hex value of `#62A559`. If I were to globalize the variables, they'd have to get different names (e.g. `--color-darkGreen` and `--color-lightGreen`).